services/src/test/java/io/scalecube/services/a/b/testing/ServiceInfoTests.java

```diff
@@ -0,0 +1,63 @@
+package io.scalecube.services.a.b.testing;
+
+import static org.junit.Assert.assertTrue;
+
+import io.scalecube.services.ServiceInfo;
+
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class ServiceInfoTests {
+
+  @Test
+  public void test_service_info(){
+    
+    Map<String,String> tags = new HashMap<>();
+    tags.put("key1", "value1");
+    tags.put("key2", "value2");
+    
+    ServiceInfo info = new ServiceInfo("service1", tags);
+    
+    String metadata = info.toMetadata();
+    System.out.println(metadata);
```

Comment:
it would be more clear if to put here instead of system.out some assertEquals and check either it equal to expected string.